app/src/main/java/org/schabi/newpipe/local/history/StatisticsPlaylistFragment.java

```diff
@@ -96,14 +97,24 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         return inflater.inflate(R.layout.fragment_playlist, container, false);
     }
 
+    @Override
+    public void setUserVisibleHint(boolean isVisibleToUser) {
```

Comment:
Redundant code: Same as with mIsVisibleToUser. This should maybe be put into `BaseFrament`.